app/lang/en.json

```diff
@@ -314,15 +326,15 @@
     "applicationNumber": "Application number",
     "viewTheLpaProgress": "View the LPA progress",
 
-    "howShouldAttorneysMakeDecisions": "How should the attorneys make decisions?",
+    "howShouldAttorneysMakeDecisions": "How do you want your attorneys to make decisions?",
     "howShouldAttorneysMakeDecisionsDetail": "Your choice here is very important as it affects how your LPA can be used. Whichever option you choose, your attorneys must always act in your best interests.",
     "getHelpMakingDecision": "Get help with making this decision",
-    "jointlyAndSeverally": "Together (Jointly and severally)",
-    "jointlyAndSeverallyHintAttorneys": "Attorneys can make decisions on their own or together. Most people choose this option because it's the most practical.",
-    "jointly": "Jointly",
-    "jointlyHintAttorneys": "Attorneys must agree unanimously on every decision, however big or small. Be careful - if one of the attorneys can no longer act, none of the other attorneys will be able to act either, unless you state otherwise in your instructions.",
-    "jointlyAndSeverallyMixed": "Jointly for some decisions, and jointly and severally for other decisions",
-    "jointlyAndSeverallyMixedHintAttorneys": "Attorneys must agree unanimously on some decisions, but can make others on their own. You must state which decisions need to be agreed unanimously.",
+    "jointlyAndSeverallyHumanised": "{{.AllAttorneysFirstNames}} can make decisions together or on their own",
+    "jointlyAndSeverallyHint": "<p>Most people choose this option as it's the most practical. It also means {{.AllAttorneysFirstNames}} can share tasks and responsibilities between them.</p><p>In legal terms, this is know as your attorneys working 'jointly and severally'.</p>",
+    "jointlyHumanised": "{{.AllAttorneysFirstNames}} must and agree make all decisions together",
```

Comment:
guessing a word is missing between "must" and "and"

Reply:
I think maybe: `{{.AllAttorneysFirstNames}} must agree and make all decisions together` (this was a copy and paste from the doc). I'll go with this.